libs/api/domains/rsk-company-info/src/lib/graphql/rsk-company-info.service.ts

```diff
@@ -84,11 +90,15 @@ export class RskCompanyInfoService {
 
   async companyInformationSearch(
     searchTerm: string,
-    first: number,
-    after = '',
+    limit: number,
+    offset = '0',
```

Comment:
Is this a valid base64 encoded offset? Probably better to call this `cursor?: string` and do `const offset = cursor ? decodeBase64(cursor) : 0`

Reply:
@eirikurn good point, fixed